client/src/components/reviews/AddReviewModal/modalHelperFunctions.jsx

```diff
@@ -0,0 +1,229 @@
+import React from 'react';
+import clsx from 'clsx';
+import { makeStyles } from '@material-ui/core/styles';
+import { Radio, Typography } from '@material-ui/core';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormControl from '@material-ui/core/FormControl';
+import FormLabel from '@material-ui/core/FormLabel';
+
+const useStyles = makeStyles({
+  root: {
+    '&:hover': {
+      backgroundColor: 'transparent'
+    }
+  },
+  icon: {
+    borderRadius: '50%',
+    width: 16,
+    height: 16,
+    boxShadow: 'inset 0 0 0 1px rgba(16,22,26,.2), inset 0 -1px 0 rgba(16,22,26,.1)',
+    backgroundColor: '#f5f8fa',
+    backgroundImage: 'linear-gradient(180deg,hsla(0,0%,100%,.8),hsla(0,0%,100%,0))',
+    '$root.Mui-focusVisible &': {
+      outline: '2px auto rgba(19,124,189,.6)',
+      outlineOffset: 2
+    },
+    'input:hover ~ &': {
+      backgroundColor: '#ebf1f5'
+    },
+    'input:disabled ~ &': {
+      boxShadow: 'none',
+      background: 'rgba(206,217,224,.5)'
+    }
+  },
+  checkedIcon: {
+    backgroundColor: '#137cbd',
+    backgroundImage: 'linear-gradient(180deg,hsla(0,0%,100%,.1),hsla(0,0%,100%,0))',
+    '&:before': {
+      display: 'block',
+      width: 16,
+      height: 16,
+      backgroundImage: 'radial-gradient(#fff,#fff 28%,transparent 32%)',
+      content: '""'
+    },
+    'input:hover ~ &': {
+      backgroundColor: '#106ba3'
+    }
+  }
+});
+
+// Inspired by blueprintjs
+function StyledRadio(props) {
+  const classes = useStyles();
+
+  return (
+    <Radio
+      className={classes.root}
+      disableRipple
+      color="default"
+      checkedIcon={<span className={clsx(classes.icon, classes.checkedIcon)} />}
+      icon={<span className={classes.icon} />}
+      {...props}
+    />
+  );
+}
+
+export function getAllCharVals(prodID) {
+  const resultsArr = [];
+  if (prodID === 18078) {
+    resultsArr.push(['60618', 'Fit', [
+      [1, 'Runs Tight'],
+      [2, 'Runs slightly tight'],
+      [3, 'Perfect'],
+      [4, 'Runs slightly Long'],
+      [5, 'Runs Long']
```

Comment:
could consider converting fit to an object and just assign the ids later to increase dry code

Reply:
I tried using an object. I have difficulty extracting it in the value= within the FormControlLabel

Reply:
sry i mean define everything but the first index that way you can reuse the array just changing the id/first index